clients/fides-js/src/lib/events.ts

```diff
@@ -65,6 +65,7 @@ export const dispatchFidesEvent = (
     const event = new CustomEvent(type, {
       detail: { ...cookie, debug, extraDetails: constructedExtraDetails },
     });
+    performance.mark(type);
```

Comment:
nit:
```suggestion
    performance && performance.mark(type);
```

I know caniuse says `performance` is everywhere, but it's probably something that could be disabled by a useragent, etc?

Reply:
gooooood call!